docs/en/datasets/detect/open-images-v7.md

```diff
@@ -67,7 +90,7 @@ Typically, datasets come with a YAML (Yet Another Markup Language) file that del
     --8<-- "ultralytics/cfg/datasets/open-images-v7.yaml"
     ```
 
-## Usage
+## Training Usage
```

Comment:
probably revert this if we are going to eliminate the predict usage above, and in other dataset pages we are directly using `Usage`.